ebos/ecloutputblackoilmodule.hh

```diff
@@ -1311,45 +1339,41 @@ public:
                 
                                 const auto flowtype = ftype(injType);
                                 const auto flowctl = fctl(ctlMode);
+                                Scalar flowPot = 0.0;
                                 if(flowtype == "Oil") //WellInjDataType::CTRLModeOil
                                 {                    
                                         if (flowctl == "RATE"){ tmp_names[1] = "ORAT"; }
                                         else { tmp_names[1] =  flowctl; }
+                                        flowPot = tmp_values[2];
```

Comment:
Here and above, this looks like it does not get the well potentials, but the actual flow rates. The calculations further down (line 1362) seem to compute some sort of injectivity index based on the flow rates (same for the productivity index on line 1233).

Is this the quantity sought? Should we not output the actual potentials at least some time? I admit the output string confuses me: "STEADY-STATE PI OR POTN. OF PREF. PH." -- when should I expect to get the potentials?

Reply:
Eclipse outputs the well potential instead of PI only when the drainage radius (DRADIUS) is set to negative in the input deck. I think we could output both. Any suggestion/preference? @jalvestad

Reply:
I would prefer the default behavior to follow the commercial simulator's reporting to avoid confusion between it and OPM Flow, from a users prospective. If we want to deviate from this we can use a switch on the keyword , WELLS=6 for example, to implement an alternative printing option.